frontend/src/components/common/form/Check.tsx

```diff
@@ -67,20 +76,43 @@ export const Check: React.FC<CheckProps> = ({
             {!!required && <span className="required">*</span>}
           </Form.Label>
         )}
-        <Form.Check
-          as={asElement}
-          name={field}
-          className={className}
-          required={required}
-          disabled={disabled}
-          custom={custom}
-          defaultChecked={checked}
-          isInvalid={!!touch && !!error}
-          {...rest}
-          value={checked}
-          placeholder={placeholder}
-          onChange={handleChange}
-        />
+        <>
```

Comment:
We've had a number of instances of radio buttons being requested for boolean flags in the DB, so I decided to stop fighting it and add generic support for that in this component.